coriolis/osmorphing/osmount/base.py

```diff
@@ -230,20 +230,15 @@ def _check_mount_fstab_partitions(
             (mountpoint, mounts[mountpoint])
             for mountpoint in sorted(mounts, key=len))
 
-        # regexes for supported fstab device references:
-        uuid_char_regex = "[0-9a-fA-F]"
-        fs_uuid_regex = (
-            "%(char)s{8}-%(char)s{4}-%(char)s{4}-"
-            "%(char)s{4}-%(char)s{12}") % {"char": uuid_char_regex}
-        fs_uuid_entry_regex = "^(UUID=%s)$" % fs_uuid_regex
-        by_uuid_entry_regex = "^(/dev/disk/by-uuid/%s)$" % fs_uuid_regex
+        # regex for supported fstab device references:
+        fs_uuid_entry_regex = "^((UUID=|/dev/disk/by-uuid/)(.+))$"
         if not mountable_lvm_devs:
             mountable_lvm_devs = []
         device_paths = self._get_device_file_paths(mountable_lvm_devs)
         for (mountpoint, details) in mounts.items():
             device = details['device']
-            if (re.match(fs_uuid_entry_regex, device) is None and
-                    re.match(by_uuid_entry_regex, device) is None):
+            fs_uuid_match = re.match(fs_uuid_entry_regex, device)
+            if fs_uuid_match is None:
```

Comment:
Positive logic is always preferable. Please change the condition to `if fs_uuid_match`.